apps/tsilhqotin-language-hub/src/Components/Pages/Home/Home.tsx

```diff
@@ -24,34 +26,58 @@ function App(): JSX.Element {
                             </div>
                         </div>
                     </div>
+
                     <div className="bottomCenter">
                         <ScrollToDiv />
                     </div>
                 </div>
 
-                <section className="flex-container">
-                    <div className="flex-child1">Cell Phone Image</div>
-
-                    <div className="flex-child2">
-                        <div id="downloadApps">
-                            <h1 className="download">DOWNLOAD</h1>
-                            <h2 className="downloadApp">Tsilhqot'in Digital Phrasebook</h2>
-                            <br />
-                            <p>
-                                The Tsilhqot'in Digital Phrasebook contains over 10,000 Tsilhqot'in
-                                terms and 1,000 Paradigms/Vocabulary Lists
-                            </p>
-                            <br />
-                            <Button className="button" variant="outlined">
-                                APP STORE
-                            </Button>
-                            <Button className="button" variant="outlined">
-                                PLAY STORE
-                            </Button>
-                        </div>
+                <div className="featuredSection">
+                    <div>
+                        <h1 id="featuredHeading">Featured</h1>
                     </div>
-                </section>
-                <section className="flex-container">hi</section>
+                    <Typography component={'span'} variant={'body2'} className="skillCard">
+                        <Card sx={{ background: 'none' }} id="alphabet2" className="featured">
+                            <Link to="/teachers" reloadDocument={true}>
+                                <CardContent className="cardBlock"></CardContent>
+                            </Link>
+                        </Card>
+                        <Card sx={{ background: 'none' }} id="phraseBook2" className="featured">
+                            <Link to="/apps" reloadDocument={true}>
+                                <CardContent className="cardBlock"></CardContent>
+                            </Link>
+                        </Card>
+
+                        <a
+                            href="https://play.google.com/store/apps/developer?id=Aaron+Plahn+%28Ts%CC%82ilhqot%E2%80%99in+National+Government%29"
```

Comment:
This could go to the config as well. For now, you might ignore pulling home from a config.